fileServer.js

```diff
@@ -17,96 +19,69 @@
 
  function send404Request(res) {
    fs.readFile('public/404.html', 'UTF-8', function(err, html) {
-<<<<<<< HEAD
      if (err) {
        res.writeHead(500);
-       res.write("Internal Server Error");
+       res.write('Internal Server Error');
+       console.log(err);
      } else {
        res.writeHead(404, {
-         'Content-type': mimeType
+         'Content-type': 'text/html'
```

Comment:
Use the mimeTypes hash, do not hard code the content type - that is why you have that hash :). Using the hash might mean you want to make a wrapper function that takes a file name as an argument and returns the proper mime type for the file e.g. ```'{ Content-Type' : mimeTypeFor(filename) }```